Tests/Unit/Repository/PhpcrOdmRepositoryTest.php

```diff
@@ -247,6 +247,41 @@ public function testMoveException()
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function testReorder()
+    {
+        $this->doTestReorder(1, false);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function testReorderToLast()
+    {
+        $this->doTestReorder(66, false);
+    }
+
+    private function doTestReorder($index, $before)
```

Comment:
It should be used .. could use a data provider, but as these are abstract tests it is a bit more complicated.